templates/suppliers/list.html.twig

```diff
@@ -62,7 +62,7 @@
                             <span class="govuk-warning-text__assistive">Important</span>
                             Some of our suppliers will not be displayed on this page
                         </strong>
-                        <p>Select <a href="https://supplierregistration.cabinetoffice.gov.uk/dps">Dynamic Purchasing Systems</a>, <a href="https://www.digitalmarketplace.service.gov.uk/g-cloud/suppliers">G-Cloud</a> or <a href="https://www.digitalmarketplace.service.gov.uk/buyers/frameworks/digital-outcomes-and-specialists-4/requirements/digital-outcomes">Digital Outcomes & Specialists</a> (DOS) to see suppliers for that type of agreement. To find out what a DPS, G-Cloud or DOS is take a look at <a href="/buy-and-supply/how-to-buy/">How to buy</a>.</p>
+                        <p>Select <a href="https://supplierregistration.cabinetoffice.gov.uk/dps">Dynamic Purchasing Systems</a>, <a href="https://www.digitalmarketplace.service.gov.uk/g-cloud/suppliers">G-Cloud</a> or <a href="https://www.digitalmarketplace.service.gov.uk/buyers/frameworks/digital-outcomes-and-specialists-4/requirements/digital-outcomes">Digital Outcomes & Specialists</a> (DOS) to see suppliers for that type of agreement. To find out what a DPS, G-Cloud or DOS is take a look at <a href="/help-and-support/how-to-buy/">How to buy</a>.</p>
```

Comment:
Is /help-and-support/how-to-buy/ the correct URL? I checked on prod and get a page not found. Or is this to be changed?

Reply:
Thanks for noticing this! This is now fixed